### DISCUSSION TITLE
Feature request: open multiple popups

### DISCUSSION THREAD

**Discussion by pjannink** [Clooos/Bubble-Card#961] [Feature requests]
I would appreciate if I could open multiple popups. Why? On a 3 column dashboard my idea is to have menu-options in column 1. Depending on the menu item I would like  to show the pop-up in column 2 or (when I have more info to show) in column 2 and 3.

**Comment by npstewart87** [comment]
You mean sort of like a mac OS finder?

It seems like this would take up a lot of screen space.

**Comment by pjannink** [comment]
No idea what mac OS Finder has to do with it.  Please see the picture.
When I select Menu1 I need one column for Items1, but Menu2 contains more information and I want to use 2 columns for that. This would require 2 popups to pop up when I select Menu2.

Hope this clarifies.

![image](https://github.com/user-attachments/assets/ad93c107-c044-4426-9e56-5fb8bb17db1a)

**Comment by npstewart87** [comment]
<img width="857" alt="image" src="https://github.com/user-attachments/assets/67d4a246-4566-4316-be03-ba5b30d127ab">

Yea, seems exactly like Mac OS Finder.

**Comment by Clooos** [comment]
Hi! I will not add this as a feature because it would be too specific and complex to do with how everything works right now.

My advice here is to have pop-ups that opens other pop-up, then when you go back the previous one will open.

Sorry for that and I hope that you understand 🙂

**Reply by npstewart87** [reply]
Agreed. This is an entirely separate integration altogether and has nothing to do with bubble card. It also would seem to have a limited user base because most people using home assistant are not trying to use it like an operating system. 

If you need a 5 level hierarchy of stored information then you probably just need to use windows.

**Comment by pjannink** [comment]
@Clooos, I accept that not all feature requests make it to the real thing.. @npstewart87 this is not “ an entirely separate integration “ and it is simply an extension idea on the bubble card concept. And dude, this is just a dashboard request and has nothing to do with using HA as an OS or whatever.  NOo idea what makes you think this way….

**Reply by npstewart87** [reply]
You could create this yourself. Bubble card is simply a uniform way to display information consistently, and hide it. Its pretty amazing overall. 

If you wanted this then you could create several template sensors and connect it to the dashboard visibility settings. Essentially you could create a tap action on a card/button that executes a script to toggle a template sensor "on/off", and the visibility settings would be set so that when sensor "A"  state is equal to "true" then it shows popup "A".  Then you would click a button on popup "A" to trigger a script that changes the state of sensor "B", and so on.

Then you would have to connect all of this to the visibility settings so that it behaves as expected, where both pop-up "A" and popup "B" would be displayed only when template sensor "A" & "B" are set to "True".

Then once you have it all set up and working, you would have to maintain it on your dash and fix it if they make a change to HA.

Seems very complicated, and certainly way out of the bubble card territory. This is very specific and even with a GUI interface in the UI, it would probably be difficult just to set it up, let alone integrate it for the masses, or program bubble card to do it.
